.github/workflows/build_cmake.yaml

```
@@ -86,3 +92,16 @@ jobs:
         continue-on-error: true
         run: |
           ctest --preset ${{ matrix.cmake_preset }}
+
+          if [[ "${{ matrix.run_test_with_coverage }}" = "true" ]]; then
+            cmake --build . --target lcov-capture --preset ${{ matrix.cmake_preset }}
+          fi
+
+      - name: Upload code coverage
+        if: ${{ matrix.run_test }} && ${{ matrix.run_test_with_coverage }}
```

actionlint (1.7.7)

if: condition "${{ matrix.run_test }} && ${{ matrix.run_test_with_coverage }}" is always evaluated to true because extra characters are around ${{ }}

(if-cond)